docs/index.md

```diff
@@ -343,7 +343,7 @@ you're done. For specific instructions on a number of common hosts, see the
 
 ## Getting help
 
-To get help with MkDocs, please use the [discussion group], [GitHub issues] or
+To get help with MkDocs, please use the [GitHub discussions], [GitHub issues] or
 the MkDocs IRC channel `#mkdocs` on freenode.
```

Comment:
I suspect we can drop the IRC channel too. I used to idle there and it was never that active. I doubt it is now.

Reply:
That's something I let @waylan decide, but yeah. The two or three times I was there was it pretty dead.

Reply:
I never look at the IRC channel. I would suggest removing any mention of it.

Reply:
Removed them.